etc/examples/paths.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+# these can be set by folks running the examples/Makefile.testing to use a different binaries than in the path
+ECHO=${ECHO:-echo}
+JQ=${JQ:-jq}
```

Comment:
jq has no platform-specific behavior, and does not need to be overridden.

Reply:
It's not installed by default on macOS, and can be installed by a user anywhere.  See https://github.com/pachyderm/pachyderm/pull/4715#discussion_r397365325